### ISSUE TITLE
Uninitialized constant Heroku::Client

### ISSUE THREAD

**Issue by kamilczerw** [logstash-plugins/logstash-input-heroku#4]
Hello I’m trying to run logstash with heroic input, but it doesn’t work. 
I tried everything and all the time I get error:

```
A plugin had an unrecoverable error. Will restart this plugin.
  Plugin: <LogStash::Inputs::Heroku app=>”my-app">
  Error: uninitialized constant Heroku::Client
  Exception: NameError
  Stack: org/jruby/RubyModule.java:2726:in `const_missing'
/home/vagrant/logstash-1.5.0/vendor/bundle/jruby/1.9/gems/logstash-input-heroku-0.1.4/lib/logstash/inputs/heroku.rb:36:in `run'
/home/vagrant/logstash-1.5.0/vendor/bundle/jruby/1.9/gems/logstash-core-1.5.0-java/lib/logstash/pipeline.rb:177:in `inputworker'
/home/vagrant/logstash-1.5.0/vendor/bundle/jruby/1.9/gems/logstash-core-1.5.0-java/lib/logstash/pipeline.rb:171:in `start_input' {:level=>:error, :file=>"logstash/pipeline.rb", :line=>"182", :method=>"inputworker"}
org/jruby/RubyModule.java:2726:in `const_missing'
/home/vagrant/logstash-1.5.0/vendor/bundle/jruby/1.9/gems/logstash-input-heroku-0.1.4/lib/logstash/inputs/heroku.rb:36:in `run'
/home/vagrant/logstash-1.5.0/vendor/bundle/jruby/1.9/gems/logstash-core-1.5.0-java/lib/logstash/pipeline.rb:177:in `inputworker'
/home/vagrant/logstash-1.5.0/vendor/bundle/jruby/1.9/gems/logstash-core-1.5.0-java/lib/logstash/pipeline.rb:171:in `start_input’
```

I’m logged in on heroku, I have java 8, I tried java 7 but the same error.

**Comment by flekmatik** [comment]
same problem for me, is this plugin working for someone? (so we could try to find root cause)

**Comment by kamilczerw** [comment]
Maybe it is connected to heroku version? Because when I changed namespace from
`require "heroku"` to `require "heroku/client"` in `https://github.com/logstash-plugins/logstash-input-heroku/blob/master/lib/logstash/inputs/heroku.rb` everything works fine.

**Comment by flekmatik** [comment]
yes, you are correct, this fixes the issue for me too

**Comment by vincent144** [comment]
Can this be tagged and released because I'm getting this error in `v 2.0.4`
